README.md

```diff
@@ -23,6 +23,22 @@ details.
 [![](https://sourcerer.io/fame/shcheklein/iterative/dvc.org/images/6)](https://sourcerer.io/fame/shcheklein/iterative/dvc.org/links/6)
 [![](https://sourcerer.io/fame/shcheklein/iterative/dvc.org/images/7)](https://sourcerer.io/fame/shcheklein/iterative/dvc.org/links/7)
 
+## Developing online
+
+Our repository has GitHub [Codespaces](https://docs.github.com/en/codespaces)
```

Comment:
I wonder if can now make Edit on Codespaces button?

Reply:
Meaning to open the particular file in your VSC codespace? That would be cool!

Keep in mind the first time you open Codespaces the environment needs to build itself, which takes a while (say 5-10 min). And when you reopen after being inactive it takes a minute or so to restart the box.

Reply:
I don't think there's a way to tell it what file to open though. At least nothing mentioned in https://docs.github.com/en/codespaces/developing-in-codespaces/creating-a-codespace.

Created https://github.community/t/codespaces-is-it-possible-to-open-on-a-specific/186839.